src/universal/modules/meeting/components/MeetingNavItem/MeetingNavItem.js

```diff
@@ -0,0 +1,153 @@
+// import React from 'react';
```

Comment:
remove? I'm probably the worst offender of leaving files like these hang out because i'll eventually go back to use them as reference. if that's whatcha wanna do, that's AOK by me, but if we can clean up, that's great, too.